packages/wallet/src/components/HeaderLogo.tsx

```diff
@@ -1,15 +1,36 @@
 import { Logo } from '@/ui/Logo'
+import { cx } from 'class-variance-authority'
 
 type HeaderLogoProps = {
   header: string
+  type?: string
 }
 
-export const HeaderLogo = ({ header }: HeaderLogoProps) => {
+export const HeaderLogo = ({ header, type }: HeaderLogoProps) => {
   return (
     <>
-      <h2 className="mb-10 text-xl text-brand-turqoise md:hidden">{header}</h2>
-      <Logo className="h-28 w-28 flex-shrink-0" />
-      <h1 className="hidden space-x-4 text-5xl font-semibold text-brand-turqoise md:mt-10 md:block">
+      <h2
+        className={cx(
+          'mb-10 text-xl md:hidden',
+          type === 'kyc' ? 'text-brand-green-3' : 'text-brand-turqoise'
+        )}
+      >
+        {header}
+      </h2>
+      <Logo
+        className={cx(
+          'h-28 w-28 flex-shrink-0',
+          type === 'kyc'
+            ? 'absolute top-8 mr-[80%] h-10 w-10 md:relative md:top-0 md:clear-none md:mx-auto md:block md:h-28 md:w-28'
+            : ''
+        )}
+      />
+      <h1
+        className={cx(
+          'hidden space-x-4 text-5xl font-semibold md:mt-10 md:block',
+          type === 'kyc' ? 'text-brand-green-3' : 'text-brand-turqoise'
+        )}
+      >
         {header}
       </h1>
     </>
```

Comment:
```suggestion
  <div className="flex flex-col-reverse items-center md:flex-col">
      <Logo className={cx('h-28 w-28 flex-shrink-0')} />
      <h1
        className={cx(
          'mb-10 block space-x-4 text-xl md:mb-0 md:mt-10 md:text-5xl md:font-semibold',
          type === 'kyc' ? 'text-brand-green-3' : 'text-brand-turqoise'
        )}
      >
        {header}
      </h1>
    </div>
```

We can do something like this and get rid of one of the header elements. ([`flex-col-reverse`](https://tailwindcss.com/docs/flex-direction#column-reversed))

Reply:
Nice solution